### DISCUSSION TITLE
大佬你好，请问这个软件的配色可否告知一下，谢谢。

### DISCUSSION THREAD

**Discussion by Hhsy083212** [readest/readest#69] [General]
大佬你好，用你写的软件看书很久了，我很喜欢其中（旧韵）这个主题的配色，看着特别舒服，平时我也写点东西，就想在word中实现相同的配色，无奈怎么都调不到一样的感觉，还用取色工具识别了好多次，可惜都不准确，可否请求您分享一下这几个主题的配色rgb呢？名称（背景，文字）这样最好了，拜托了，我实在太喜欢你写的这个软件了。
还有你这个默认的字体serif是宋体吗，word里也没有。
配色+字体，能否打个包发我邮箱一份？拜托了大佬，这年头愿意为windows写软件的人不多了，能有这么舒心的应用，再次感谢您的作品。
期待受到您的邮件。

yang083212@gmail.com

感谢，期待。

**Comment by chrox** [comment]
Transferred to discussion.

**Reply by Hhsy083212** [reply]
大佬你好，确切的说是神色模式下的旧韵主题，我刚才发现同一个主题，浅色模式和深色模式还不一样。

**Comment by chrox** [comment]
感谢你的支持。“旧韵”的主题配色英文叫“Sepia”，各种颜色的翻译对应可以在这里找到：https://github.com/readest/readest/blob/main/apps/readest-app/public/locales/zh-CN/translation.json#L92 
色值的定义在这里：https://github.com/readest/readest/blob/main/apps/readest-app/src/styles/themes.ts#L58-L61

serif 是西文排版中的衬线字体，衬线字体有很多种，宋体也算是一种衬线字体。

**Comment by chrox** [comment]
```
{
    name: 'sepia',
    label: _('Sepia'),
    colors: {
      light: generateLightPalette({ fg: '#5b4636', bg: '#f1e8d0', primary: '#008b8b' }),
      dark: generateDarkPalette({ fg: '#ffd595', bg: '#342e25', primary: '#48d1cc' }),
    },
},
```

这里面定义了深色和浅色的前景色：fg、背景色：bg和原色：primary。其他图层的颜色是系统根据一定规则生成的，我这里也没有现成的色值。你如果右键查看页面审查元素可以看到页面中每个元素的准确颜色。

**Comment by chrox** [comment]
![image](https://github.com/user-attachments/assets/87955a8d-391a-4722-91ed-266e757d323d)
你如果没选择其他字体，默认正文使用衬线字体：Bitter，这个字体可以在Google Fonts下载：https://fonts.google.com/specimen/Bitter
